pom.xml

```diff
@@ -7,8 +7,23 @@
   <version>1.14-SNAPSHOT</version>
   <packaging>maven-plugin</packaging>
 
-  <name>maven-license-plugin Maven Plugin</name>
-  <url>http://maven.apache.org</url>
+  <name>License Maven Plugin</name>
+  <url>https://github.com/cloudbees/maven-license-plugin</url>
+  <description>Maven plugin to process license information.</description>
+  <licenses>
+    <license>
+      <name>Apache 2.0 License</name>
+      <url>https://opensource.org/licenses/Apache-2.0</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+  <developers>
+    <developer>
+      <id>jglick</id>
```

Comment:
Not really used for anything but I guess as a contact in case there is a problem on Central.